src/app/starredentries/starredentries.component.scss

```diff
@@ -58,6 +58,11 @@
   text-decoration: none;
   transition: 0.4s color;
 }
+.notebook-icon {
```

Comment:
Why does this icon need special styling? Can't it use the same styling as the other icons on the page? Maybe it does, I didn't look at it closely, but it seems like it shouldn't.